app/javascript/readingList/readingList.jsx

```diff
@@ -92,220 +89,153 @@ export class ReadingList extends Component {
       body: JSON.stringify({ current_status: statusView }),
       credentials: 'same-origin',
     });
-    const newItems = readingListItems;
+
+    const t = this;
+    const newItems = items;
     newItems.splice(newItems.indexOf(item), 1);
     t.setState({
       archiving: true,
-      readingListItems: newItems,
-      totalReadingList: totalReadingList - 1,
+      items: newItems,
+      totalCount: totalCount - 1,
     });
+
+    // hide the snackbar in a few moments
     setTimeout(() => {
       t.setState({ archiving: false });
-    }, 1800);
+    }, 1000);
```

Comment:
1.8 seconds it's a lot for the human eye to see a snackbar that's supposed to disappear, so I lowered it to 1 second